plugins/inputs/icinga2/README.md

```diff
@@ -24,10 +24,18 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 [[inputs.icinga2]]
   ## Required Icinga2 server address
   # server = "https://localhost:5665"
-  
-  ## Required Icinga2 object type ("services" or "hosts")
+  ## Collected Icinga2 objects ("services", "hosts")
+  ## Specify at least one object to collect from /v1/objects endpoint.
+  # objects = ["services"]
+
+  ## deprecated in 1.26.0; use the `objects` array-based option
   # object_type = "services"
```

Comment:
Can we please remove this lines. I think having the _recommended_ way of configuring the plugin in `sample.conf` is the right thing to do.